gum/backend-darwin/gumobjcapiresolver.c

```diff
@@ -176,8 +176,16 @@ gum_objc_api_resolver_enumerate_matches (GumApiResolver * resolver,
   gboolean carry_on;
   GHashTable * visited_classes;
   GumObjcClassMetadata * klass;
-
-  g_regex_match (self->query_pattern, query, 0, &query_info);
+  
+  gchar * query_copy = (gchar *) query;
```

Comment:
Variable declarations should be in chronological order, and at the beginning of the scope. More on the style here:
https://github.com/frida/frida/blob/master/CONTRIBUTING.md